source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.h

```diff
@@ -29,7 +29,7 @@ class CreateMaterialXNodeCmd : MPxCommand
     /// @name Maya API methods
     /// @{
     MStatus doIt(const MArgList&) override;
-    bool isUndoable() { return false; }
+    bool isUndoable() const override { return false; }
```

Comment:
Strange this would be missed in a local build even on Windows.

Reply:
Clang performs an extra check with `-Woverloaded-virtual`